### PR TITLE
feat(frontend): BtcSendTokenWizard component

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Create BtcSendTokenWizard component.

### CODE REVIEW

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		btcAddressMainnet,
+		btcAddressRegtest,
+		btcAddressTestnet
+	} from '$lib/derived/address.derived';
+	import { WizardStepsSend } from '$lib/enums/wizard-steps';
+	import type { NetworkId } from '$lib/types/network';
+	import { isNetworkIdBTCRegtest, isNetworkIdBTCTestnet } from '$lib/utils/network.utils';
+	import { decodeQrCode } from '$lib/utils/qr-code.utils';
+
+	export let currentStep: WizardStep | undefined;
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:19]: not sure why the `currentStep` is exposed?

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	export let networkId: NetworkId | undefined = undefined;
+	export let destination = '';
+	export let amount: number | undefined = undefined;
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:22]: ditto

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	export let sendProgressStep: string;
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:23]: ditto

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+{:else if currentStep?.name === WizardStepsSend.QR_CODE_SCAN}
+	<SendQrCodeScan
+		expectedToken={$sendToken}
+		bind:destination
+		bind:amount
+		{decodeQrCode}
+		on:icQRCodeBack
+	/>
+{:else}
+	<slot />
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:62]: What's the purpose of this `slot`?

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:19]: `currentStep` will be controlled by the parent component called `BtcSendModal` (the next PR). This is how it's done in `IcSendModal` - https://github.com/dfinity/oisy-wallet/blob/941024a552be4cb4378c9ca3862df7d986cf59e6/src/frontend/src/icp/components/send/IcSendModal.svelte#L55

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte (hunk 2 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:22]: Same with `amount` - https://github.com/dfinity/oisy-wallet/blob/941024a552be4cb4378c9ca3862df7d986cf59e6/src/frontend/src/icp/components/send/IcSendModal.svelte#L25

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:19]: Ah gotcha! I think I reviewed it thinking we were one level up here. You can probably ignore all my comments.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

Note: the `source` and `networkId` properties have been discussed "offline" with @DenysKarmazynDFINITY

**Pull request merged**